src/matter/cluster/OccupancySensingCluster.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
```

Comment:
Comment blocks starting with /** are reserved for jsdoc, so should be before a class / method / parameter.

Comment blocks which are not jsdoc, like this one, should start with /* (one star)

Reply:
@JimBuzbee You "just need to move" this comment block above the "...=Cluster" öine below ... then all is correct. It in fact is the documentation belonging there